.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,30 @@
+name: Documentation
+description: Report problems and suggest ideas for the documentation
+labels: [docs]
+
+body:
+  - type: markdown
+    attributes:
+      value: Before opening a new issue, please search for duplicate or closed issues.
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: Skip to "other" if you do not see your issue type here.
+      options: [improvement, mistake]
+
+  - type: input
+    attributes:
+      label: Kind of issue (other)
+      description: Only fill out if your issue type is not in the dropdown above.
+
+  - type: textarea
+    attributes:
+      label: Description
```

Comment:
I would like to add a `placeholder`, though I'm not sure what to put in :thinking: Maybe we can put a generic example, something like:

```suggestion
      label: Description
      placeholder: |
        Example: The documentation doesn't cover my use case of the NPM package ...
```

Reply:
Do we need the `Example:` here? I feel like that's already implied if it's a placeholder.

Reply:
I agree it's not totally necessary, I just think textboxes with placeholders look better. We could do something similar to GitHub (:arrow_down:), maybe just _"Add a description"_. In any case, if we can't come to an agreement here we can leave it for now :+1: 

![image](https://user-images.githubusercontent.com/3742559/129859638-3e148859-d52b-48a3-a99a-723784a9a76c.png)


> Also would you like me to add placeholders for all the fields?

If we do it here, I'd say we do it everywhere.

Reply:
Oh no, I prefer placeholders too, I'm just asking if we literally need to include the text "Example:".

Reply:
Ah, I took that from the snippets in the GitHub documentation for issue forms, e.g.

```yaml
- type: textarea
  attributes:
    label: Operating System
    description: What operating system are you using?
    placeholder: Example: macOS Big Sur
    value: operating system
```

itt seemed kinda nice but I don't have any strong opinions on whether to include "Example:" or not